src/components/Modeler.vue

```diff
@@ -586,6 +588,35 @@ export default {
         }
       });
     },
+
+    getBoundaryEvents(process) {
+      return process.flowElements
+        ? process.flowElements.filter(({$type}) => $type === 'bpmn:BoundaryEvent')
+        : [];
+    },
+    createBoundaryEvent(definition) {
+      let boundaryEvent = this.moddle.create('bpmn:BoundaryEvent', {
+        ...definition,
+        cancelActivity: definition.get('cancelActivity'),
+        attachedToRef: definition.get('attachedToRef'),
+      });
```

Comment:
I think we can reuse the same function defined in `/src/components/nodes/boundaryEvent/index.js` to create this definition:
```js
import boundaryEventConfig from './nodes/boundaryEvent';
...
const boundaryEvent = boundaryEventConfig.definition(this.moddle, this.$t);
```

Though we'd have to be careful to re-assign the properties from the original `definition`. The benefit here is that now we have a single place where the boundary event definition is declared.

**Note:** Since `boundaryEvent` isn't reassigned, you can use `const` instead of `let`.